cmd/config.go

```diff
@@ -92,14 +92,14 @@ $ %s cfg list`, appName, defaultHome, appName)),
 			case yml && jsn:
 				return fmt.Errorf("can't pass both --json and --yaml, must pick one")
 			case jsn:
-				out, err := json.Marshal(ConfigToWrapper(a.Config))
+				out, err := json.Marshal(a.Config)
```

Comment:
We were previously using the config wrapper here and we still use it on L102 below so we may want this to be the wrapper still also.